spec/requests/publishers/vacancies/copies_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "copy" do
+  let(:publisher) { create(:publisher) }
+  let(:organisation) { build(:school) }
+
+  before do
+    # rubocop:disable RSpec/AnyInstance
+    allow_any_instance_of(Publishers::BaseController).to receive(:current_organisation).and_return(organisation)
+    # rubocop:enable RSpec/AnyInstance
+    sign_in(publisher, scope: :publisher)
+  end
+
+  after { sign_out(publisher) }
+
+  describe "POST /copy" do
+    let(:vacancy) { create(:vacancy, :with_application_form, :with_supporting_documents, organisations: [organisation]) }
+
+    it "doesn't crash" do
+      post organisation_job_copy_path(vacancy.id)
+      expect(response).to have_http_status(:found)
```

Comment:
Maybe we could add some additional assertions here to try to get more value out of the test, perhaps something like:

`expect { post organisation_job_copy_path(vacancy.id) }.to change(Vacancy, :count).by(1)`

Reply:
That was my intention, but didn't re-visit after Brandon checked it over...